src/frontend/src/reducers/neighborhoods/neighborhoodsSlice.ts

```diff
@@ -18,18 +16,6 @@ export const neighborhoodSlice = createSlice({
     name: "neighborhoods",
     initialState,
     reducers: {
-        setNeighborhoods: (
-            state,
-            { payload: neighborhoods }: { payload: Neighborhoods }
-        ) => {
-            state.neighborhoods = neighborhoods;
-        },
-        setNeighborhoodBounds: (
-            state,
-            { payload: neighborhoodBounds }: { payload: NeighborhoodBounds }
-        ) => {
-            state.neighborhoodBounds = neighborhoodBounds;
-        },
```

Comment:
Took these out because I realized they're set via the `exraReducers` below and there's no scenario where we'd be updating neighborhoods/bounds data from the frontend.